builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -263,14 +263,17 @@ def loadServiceConfigurationData() {
 			sh "sed -i -- 's/{conf-user-pool-id}/${config_loader.AWS.COGNITO.USER_POOL_ID}/g' ./config/dev-config.json"
 			sh "sed -i -- 's/{conf-client-id}/${config_loader.AWS.COGNITO.CLIENT_ID}/g' ./config/dev-config.json"
 			sh "sed -i -- 's/{conf-region}/${region}/g' ./config/dev-config.json"
+      sh "sed -i -- 's/{jazz_admin}/${config_loader.JAZZ.ADMIN}/g' ./config/dev-config.json"
 
 			sh "sed -i -- 's/{conf-user-pool-id}/${config_loader.AWS.COGNITO.USER_POOL_ID}/g' ./config/stg-config.json"
 			sh "sed -i -- 's/{conf-client-id}/${config_loader.AWS.COGNITO.CLIENT_ID}/g' ./config/stg-config.json"
 			sh "sed -i -- 's/{conf-region}/${region}/g' ./config/stg-config.json"
+      sh "sed -i -- 's/{jazz_admin}/${config_loader.JAZZ.ADMIN}/g' ./config/stg-config.json"
```

Comment:
still seeing indentation issues

Reply:
fixed the indentation issue
it's looking fine in the editor -

Reply:
The indentation is looking correct   in my IDE (VScode) and is looking correct in the repo as well , not sure why there is a mismatch in the pull request 
https://github.com/hks93/jazz/blob/Admin-API/builds/jazz-build-module/service-configuration-data-loader.groovy

Reply:
It looks you have a mix of tab delimiter and space delimiter which is making it look right on your IDE but will be different when it comes to a diff IDE. I fixed it for lines 92, 289, 294 and 299 => copy a line above or below that looks correct (including the white space/indentation) and paste it where you have incorrect indentation. Now update the code and now your indentation issue will be fixed.